snapcraft/plugins/catkin.py

```diff
@@ -105,6 +105,7 @@ def env(self, root):
                 root,
                 self.options.rosversion),
             'ROS_MASTER_URI=http://localhost:11311',
```

Comment:
That maybe it eventually is expected to run on a different port. But I'm not sure that is configurable.